src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -215,6 +215,59 @@
       <TranslationFile>$(MIEngineRoot)loc\LCL\{Lang}\lib\OpenFolderSchema.json.lcl</TranslationFile>
     </FilesToLocalize>
   </ItemGroup>
+  <Target Name="DropLocalizeDir" AfterTargets="Localize;DropFiles" Condition="'$(LocalizationEnabled)'=='true'">
+    <PropertyGroup>
+      <LocalizedDropDir>$(DropRootDir)\loc\</LocalizedDropDir>
+      <LocalizeBinDir>$(OutDir)localize\</LocalizeBinDir>
+    </PropertyGroup>
+    <ItemGroup>
+      <Locale Include="$(LocalizeBinDir)ENU\*.json">
```

Comment:
nit: Since each item represents a file to copy rather than a language, we might want to give this a better name. `LocalizedFileToDrop` maybe?